src/ucp/tag/rndv.c

```diff
@@ -276,11 +345,12 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_progress_rma_get_zcopy, (self),
     state = rndv_req->send.state.dt;
     ucp_dt_iov_copy_uct(iov, &iovcnt, max_iovcnt, &state, rndv_req->send.buffer,
                         ucp_dt_make_contig(1), length);
+    ucp_rndv_get_fixup_iov(iov, rndv_req);
```

Comment:
need to extend ucp_dt_iov_copy_uct() to get memh_index

Reply:
extended